front/app/modules/commercial/content_builder/admin/components/ContentBuilderSettings/index.tsx

```diff
@@ -56,7 +56,10 @@ const ContentBuilderSettings = () => {
     actions.selectNode();
   };
 
-  return selected && isEnabled && selected.id !== ROOT_NODE ? (
+  return selected &&
+    isEnabled &&
+    selected.id !== ROOT_NODE &&
+    selected.name !== 'Box' ? (
     <StyledBox
```

Comment:
Fixes the issue with a "default" settings box that opens when clicking divs or Boxes